js/models/userMd.js

```diff
@@ -30,7 +30,7 @@ module.exports = Backbone.Model.extend({
     blocked: [], //a list of guids to block (LIST of 40 character hex strings)
     blocked_guids: [], // the same as blocked. The API returns blocked_guids, but expects blocked.
     libbitcoin_server: "", //the server address (url string)
-    ssl: true, //use ssl on the openbazaar server (�True� or �False�)
+    ssl: false, //use ssl on the openbazaar server (�True� or �False�)
```

Comment:
Well I'm going to start making it use the values from the settings, so yeah it would turn on ssl.